service/src/main/java/greencity/service/CommentServiceImpl.java

```diff
@@ -355,4 +374,19 @@ public void delete(Long id, UserVO userVO) {
 
         commentRepo.save(comment);
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @Transactional(readOnly = true)
```

Comment:
Why do we need `Transactional` here?

Reply:
I think for optimization of performance by "readOnly = true"